GiniVision/Classes/Networking/DocumentsService.swift

```diff
@@ -17,6 +17,7 @@ final class DocumentsService: DocumentServiceProtocol {
     
     init(sdk: GiniSDK) {
         self.giniSDK = sdk
+        _ = self.getSession()
```

Comment:
Not really a fan of that. Methods shouldn't just be call for their side effects.
We do it in tests when we call `_ = controller.view` but that's different - it's not production code.